themes/default/content/blog/oidc-blog/index.md

```diff
@@ -0,0 +1,90 @@
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - pat-gavlin
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+    - pulumi-releases
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+After [launching Pulumi Deployments](/blog/pulumi-deployments) a few months ago and receiving a ton of community feedback (thank you!) we are launching today an integration with OpenID Connect (OIDC) to enable temporary credentials and granular access controls. We would like to thank GitHub user [eriklz](https://github.com/eriklz) for creating the [original request](https://github.com/pulumi/service-requests/issues/144) for this functionality.
+
+<!--more-->
+
+[OpenID Connect (OIDC)](https://openid.net/connect) is an identity layer built on top of the OAuth 2.0 framework. It allows third-party applications to verify the identity of the end-user and to obtain basic user profile information. OIDC uses JSON web tokens (JWTs), which you can obtain using flows conforming to the OAuth 2.0 specifications.
+
+## Overview
+
+The main benefits of using the OIDC integration for Pulumi Deployments are as follows:
+
+1. **Temporary credentials**: You can move away from long term credentials within your organization to leverage temporary credentials that are generated every time a Deployment runs. The Pulumi Service issues a new OIDC token specific to that deployment and then it automatically expires, meaning you do not duplicate your cloud credentials and instead can rotate credentials for each deployment.
+2. **Granular with your access control**: The Pulumi Service will generate the OIDC token, a signed JWT that contains information about the deployment, to be exchanged for cloud credentials. In AWS for example, you can configure our OIDC integration to only use temporary credentials if that token is associated with a particular Pulumi stack or Pulumi organization. This enables you to follow the principle of least privilege access and scope access down to only what is needed.
+
+## Set up
+
+### Calling the Deployments REST API
+
+You can call the [Deployments REST API](/docs/reference/deployments-rest-api) and use OIDC instead of cloud secrets. Here is an example of how that might look:
+
+![Deployment REST API payload before and after](deployment-api-payload.png)
+
+### Enabling OIDC for your Stack
```

Comment:
Title might clarify that this is only required for Push to Deploy. There is no configuration required for API usage other than specifying the appropriate params in the POST payload.